<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundfontEditorLeft</class>
 <widget class="QWidget" name="SoundfontEditorLeft">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>203</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="12" column="0" colspan="2">
    <widget class="FilterFlow" name="filterArticulation" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="labelDate">
     <property name="text">
      <string notr="true">labelDate</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="labelTimbre">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Timbre</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="labelAuthor">
     <property name="text">
      <string notr="true">labelAuthor</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="FilterFlow" name="filterSampleSource" native="true"/>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QLabel" name="labelGenre">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Genre</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="iconLicense">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLabel" name="labelArticulation">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Articulation</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="2">
    <widget class="FilterFlow" name="filterGenre" native="true"/>
   </item>
   <item row="18" column="0" colspan="2">
    <widget class="FilterTag" name="filterTag" native="true"/>
   </item>
   <item row="17" column="0" colspan="2">
    <widget class="QLabel" name="labelTags">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tags</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="2">
    <widget class="FilterFlow" name="filterMidiStandard" native="true"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="SmallComboBox" name="comboCategory"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="iconDate">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="iconWebsite">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="SmallComboBox" name="comboLicense"/>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="labelCategory">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Category</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="StyledLineEdit" name="lineWebsite">
     <property name="placeholderText">
      <string>Website...</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="labelSampleSource">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Sample source</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="FilterFlow" name="filterTimbre" native="true"/>
   </item>
   <item row="19" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>19</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="iconAuthor">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="15" column="0" colspan="2">
    <widget class="QLabel" name="labelMidiStandard">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>MIDI standard</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilterFlow</class>
   <extends>QWidget</extends>
   <header>filterflow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FilterTag</class>
   <extends>QWidget</extends>
   <header>filtertag.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StyledLineEdit</class>
   <extends>QLineEdit</extends>
   <header>styledlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>SmallComboBox</class>
   <extends>QComboBox</extends>
   <header>smallcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
